soundbay/conf/optim/jasco_resnet_50_finetune.yaml

```
@@ -0,0 +1,12 @@
+# @package _global_
+optim:
+  epochs: 100
+  optimizer:
+    _target_: torch.optim.SGD
+    lr: 0.001
+    momentum: 0.9
+    weight_decay: 0.0001
+  scheduler:
+    _target_: torch.optim.lr_scheduler.ExponentialLR
+    gamma: 0.95
+  num_unfrozen: 0 # 0 = No freeze, 1 = freeze FC,   2 = freeze FC+block4,   ...
```

```suggestion
  num_unfrozen: 0 # 0 = No freeze, 1 = unfreeze FC,   2 = unfreeze FC+block4,   ...
```
And as suggested before, I think `num_unfrozen: 0` should mean "all are frezen"